src/Umbraco.Core/Exceptions/DataOperationException.cs

```diff
@@ -1,21 +1,94 @@
 ﻿using System;
+using System.Runtime.Serialization;
 
 namespace Umbraco.Core.Exceptions
 {
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    /// <seealso cref="System.Exception" />
+    [Serializable]
     internal class DataOperationException<T> : Exception
     {
+        /// <summary>
+        /// Gets the operation.
+        /// </summary>
+        /// <value>
+        /// The operation.
+        /// </value>
         public T Operation { get; private set; }
```

Comment:
As `T` could be any type, serialization can fail if the passed in type isn't serializable.

Reply:
As the exception is currently internal and the only type used for `T` is `MoveOperationStatusType`, we might just remove the generics, right?